include.js

```diff
@@ -1814,6 +1814,13 @@ let cards = [
     imageLink: './Art/Sayed-Husain/hover.gif',
     author: 'Sayed-Husain',
     githubLink: 'https://github.com/Sayed-Husain'
+  },
+  {
+    artName: 'Circle Hover Animation',
+    pageLink: './Art/subhayudutta/index.html',
+    imageLink: './Art/subhayudutta/style.css',
```

Comment:
This needs to be a link to a `.gif` or `.png`.

Please

- [ ] make a screenshot or `.gif` of your animation
- [ ]  fix the image link entry